docs/building.md

```diff
@@ -33,6 +33,16 @@ You can use brew as shown below
 brew install bdw-gc
 ```
 
+### OS X Yosemite
+
+OS X Yosemite uses outdated version of `clang` so you might have to install a newer version via brew:
+
+```
+brew install homebrew/versions/llvm37
```

Comment:
Is this the latest version of llvm available on homebrew atm? Don't they have 3.8 too?